src/cpp/core/include/core/system/Process.hpp

```diff
@@ -59,14 +59,19 @@ struct ProcessOptions
    ProcessOptions()
 #ifdef _WIN32
       : terminateChildren(false),
```

Comment:
@jjallaire no change needed until I do the win32 work; the cols and rows fields are only in the non-Win32 build